packages/core-tester-cli/lib/commands/transfer.js

```diff
@@ -125,7 +126,7 @@ module.exports = class TransferCommand extends Command {
       transactions.push(transaction)
 
       if (log) {
-        logger.info(`${i} ==> ${transaction.id}, ${wallet.address} (fee: ${transaction.fee})`)
+        logger.info(`${i} ==> ${transaction.id}, ${this.options.recipient || wallet.address} (fee: ${transaction.fee})`)
```

Comment:
actually, instead of `this.options.recipient || wallet.address` you should be able to do `transaction.recipientId`

Reply:
That'll work